nat/mapping/port_mapping.go

```diff
@@ -72,7 +72,12 @@ func mapPort(m portmap.Interface, c chan struct{}, protocol string, extPort, int
 		}
 	}()
 	for {
-		addMapping(m, protocol, extPort, intPort, name, publisher)
+		err := addMapping(m, protocol, extPort, intPort, name, publisher)
```

Comment:
Why we need another error printouts? We already have it inside addMapping..

Reply:
Yeah, removed this.